src/main/webapp/js/webrtc_adaptor.js

```diff
@@ -256,7 +256,9 @@ function WebRTCAdaptor(initialValues)
 			});
 		}
 		else {
-			stream.addTrack(audioStream.getAudioTracks()[0]);
+			if(typeof audioStream != "undefined"){
```

Comment:
Is that working in the same error?

I wonder if this is a case that audioStream exist(not undefined) but it does not have audio tracks.

Reply:
Sorry for the wrong information. As I see in my test, audioStream is `undefined` when mediaConstraint audio false parameter.

Reply:
@golgetahir could you also please add this request to your ecma 6 branch?